yang/IETF-04/ietf-l3vpn-ntw.yang

```diff
@@ -1893,7 +1939,7 @@ module ietf-l3vpn-ntw {
                     }
                     leaf-list tree-flavor {
                       type identityref {
-                        base vpn-common:multicast-tree-type;
```

Comment:
Multicast tree type should be kept in the common module.

Reply:
solved 951497867189b7da175218986a06ab8dc66ac80f